atom/browser/api/atom_api_web_contents.cc

```diff
@@ -2554,7 +2554,11 @@ v8::Local<v8::Value> WebContents::TabValue() {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
 
   std::unique_ptr<base::DictionaryValue> value(
-    ExtensionTabUtil::CreateTabObject(web_contents())->ToValue().release());
+      ExtensionTabUtil::CreateTabObject(
+          web_contents(), ExtensionTabUtil::ScrubTabBehaviour::kScrubTab,
```

Comment:
This should be `kDontScrubTab` or browser-laptop will not have url, title and favicon_url info when calling `tabValue`